templates/nginx.conf

```diff
@@ -4,9 +4,18 @@ upstream pretalx_{{ pretalx_instance_identifier }}_server {
 
 proxy_cache_path /tmp/nginx-pretalx-{{ pretalx_instance_identifier }} levels=1:2 keys_zone=pretalx_static_{{ pretalx_instance_identifier }}:10m inactive=60m max_size=250m;
 
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name {{ pretalx_domain }}{% if pretalx_alternate_domains %} {{ pretalx_alternate_domains }}{% endif %};
+    return 301 https://{{ pretalx_domain }}$request_uri;
```

Comment:
I'm not sure this is strictly correct – this will always redirect to the main domain on HTTPS, instead of the current request hostname.